content/docs/composition-vs-inheritance.md

```diff
@@ -1,22 +1,22 @@
 ---
 id: composition-vs-inheritance
-title: Composition vs Inheritance
+titlu: Composition vs Inheritance
 permalink: docs/composition-vs-inheritance.html
 redirect_from:
   - "docs/multiple-components.html"
-prev: lifting-state-up.html
-next: thinking-in-react.html
+anterior: lifting-state-up.html
+urmator: thinking-in-react.html
 ---
 
-React has a powerful composition model, and we recommend using composition instead of inheritance to reuse code between components.
+React are un model de compoziție puternic și vă recomandăm să utilizați compoziția în locul moștenirii pentru a reutiliza codul, între componente.
 
-In this section, we will consider a few problems where developers new to React often reach for inheritance, and show how we can solve them with composition.
+În această secțiune, vom analiza câteva probleme în care dezvoltatorii novici în React aleg adesea moștenirea și vom arata cum le putem rezolva prin compoziție.
 
-## Containment {#containment}
+## Izolare {#containment}
 
-Some components don't know their children ahead of time. This is especially common for components like `Sidebar` or `Dialog` that represent generic "boxes".
+Unele componente nu își cunosc copiii înainte de timp. Acest lucru este valabil mai ales pentru componente precum `Sidebar` sau` Dialog` care reprezintă casete generice.
 
-We recommend that such components use the special `children` prop to pass children elements directly into their output:
+Recomandăm ca astfel de componente să utilizeze proprietățile specifice pentru `copii` astfel încât sa transmita elementele de copii direct în producția lor:
```

Comment:
This phrase refers to the 'children' props, in translation meaning: proprietatea speciala [children](https://reactjs.org/docs/jsx-in-depth.html#children-in-jsx)